### PR TITLE
3 list space

### PR DESCRIPTION
**Pull request by Managram**
there's no factory girl in this, but we can fix that easily

connects #3

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -12,16 +12,23 @@ class MakersBnb < Sinatra::Base
   end
 
   post '/space/new' do
-    space = Space.create(name: params[:name],
+    @space = Space.create(name: params[:name],
```
Comment (junyuanxue) [inline app/app.rb]: can just use local var instead of instance var

File: spec/features/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+feature 'displays all spaces' do
+  scenario 'the page displays spaces in the database' do
+    register_space
+    expect(page).to have_text "newspace"
```
Comment (junyuanxue) [inline spec/features/index_spec.rb]: use "new space" instead of "newspace", or use Factory Girl objects

**Comment by junyuanxue** [comment]
:shipit:

**Comment by hawksdoves** [comment]
:shipit:

**Pull request merged**